.gitignore

```diff
@@ -1,5 +1,28 @@
+# raw images are too large for Github so it is ignored
 0.download-data/Images
-1.preprocessing-data/IC_Corrected_Images
+
+# ignore pycache 
 __pycache__
+
+# test files that are not relevant to the pipelines
 1.preprocessing-data/test.ipynb
 example.csv
+
+# old raw images that were being used for analysis but is not used 
+0.download-data/Old_Images
+
+# old IC images from previous data that is not used
+1.preprocessing-data/Old_IC_Images
+
+# IC corrected that totalling 9.5 GB -> too large for GitHub
+1.preprocessing-data/IC_Corrected_Images
+
+# .txt file with output from IC for reference
+1.preprocessing-data/IC_log.txt
+
+# test jupyter notebook for running CellProfiler
+2.cellprofiler_processing/test.ipynb
+
+# ignore the output files as they are too large
+2.cellprofiler_processing/CellProfiler_output/localhost220512140003_KK22-05-198.sqlite
+2.cellprofiler_processing/CellProfiler_output/localhost220513100001_KK22-05-198_FactinAdjusted.sqlite
```

Comment:
Could these lines be condensed into an ignore pattern for `*.sqlite` files?

Reply:
Yup!